api.md

```diff
@@ -97,6 +103,43 @@ Response
 }
 ```
 
+
+#### `DELETE /api/v2/device/:deviceID` - deletes the device from your tailnet
+Deletes the provided device from your tailnet. 
+The device must belong to your tailnet.
```

Comment:
"your tailnet" again implies you can only have one, which is currently true but not something I believe we want to assume going forward